systest/tests/timeskew_test.go

```diff
@@ -15,8 +15,8 @@ import (
 
 func TestShortTimeskew(t *testing.T) {
 	const (
-		enableSkew = 10
-		stopSkew   = enableSkew + 3
+		enableSkew = 9
```

Comment:
can you add comments (like your PR description)? otherwise its easy to change it to a different value for other devs